bin/Powheg/examples/ggHH_quark-mass-effects_withJHUGen_PDF4LHC_13TeV/ggHH.powheg.input

```diff
@@ -0,0 +1,82 @@
+!                      2: approximated full theory (FTapprox)
+!                      3: full theory
+
+hmass 125            ! Higgs boson mass
+topmass 173          ! top quark mass
+hdecaymode -1        ! PDG code for Higgs boson decay products (it affects only the SMC)
+!                      allowed values are:  
+!                      0 all decay channels open
+!                      1-6 d dbar, u ubar,..., t tbar (as in HERWIG)
+!                      7-9 e+ e-, mu+ mu-, tau+ tau-
+!                      10  W+W-
+!                      11  ZZ
+!                      12  gamma gamma      
+!                      -1  all decay channels closed
+
+! general parameters:
+numevts  NEVENTS     ! number of events to be generated
+ih1         1        ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2         1        ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 6500d0        ! energy of beam 1
+ebeam2 6500d0        ! energy of beam 2
+! To be set only if using LHA pdfs
+lhans1 306000        ! pdf set for hadron 1 (LHA numbering)
+lhans2 306000	     ! pdf set for hadron 2 (LHA numbering)
+
+! Parameters to allow or not the use of stored data
+use-old-grid    1    ! if 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound  1    ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+!
+ncall1  10000        ! number of calls for initializing the integration grid
+itmx1       3        ! number of iterations for initializing the integration grid
+ncall2  10000        ! number of calls for computing the integral and finding upper bound
+itmx2       3        ! number of iterations for computing the integral and finding upper bound
+foldcsi     1        ! number of folds on csi integration
+foldy       1        ! number of folds on  y  integration
+foldphi     1        ! number of folds on phi integration
+nubound  5000        ! number of bbarra calls to setup norm of upper bounding function
+
+! OPTIONAL PARAMETERS
+
+! scale settings:
+facscfact   1d0      ! (default 1d0) fac scale factor: mufact = muref * facscfact
+renscfact   1d0      ! (default 1d0) ren scale factor: muren  = muref * renscfact
+
+! Born, plots and powheg strategy:
+testplots         1  ! (default 0, do not) do NLO and PWHG distributions
+fastbtlbound      1  ! fast evaluation of Btilde upper bounds
+storemintupb      1  ! store function calls in binary file for more precise generation
+ubexcess_correct  1  ! corrects for error in upper bound estimate
+storeinfo_rwgt    1  ! store info to allow for reweighting
+softtest          0  ! no check of the soft limit in the real
+colltest          0  ! no check of the collinear limit in the real
+
+! PARAMETERS needed for ggHH
+ptHHcut     0.001d0
+ptHHcut_CT  0.001d0
+
+! PARAMETERS for parallel runnning
+! -- non parallel (for testing purposes only)
+iseed SEED
+
+! -- parallel
+manyseeds 1          ! Used to perform multiple runs with different random
```

Comment:
I am afraid this is deprecated/unused in cms. Any reason to use it?

Reply:
No special reason, the lines were removed